addons/account/migrations/12.0.1.1/post-migration.py

```diff
@@ -0,0 +1,174 @@
+            OR option = 'last_day_current_month'
+        """
+    )
+    openupgrade.map_values(
+        cr,
+        openupgrade.get_legacy_name('option'),
+        'option',
+        [('fix_day_following_month', 'after_invoice_month'),
+         ('last_day_following_month', 'day_following_month'),
+         ('last_day_current_month', 'day_current_month'),
+         ],
+        table='account_payment_term_line', write='sql')
+
+
+def map_account_tax_type_tax_use(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_tax
+        SET type_tax_use = 'adjustment'
+        WHERE type_tax_use = 'none' AND %s = TRUE
+        """, (AsIs(openupgrade.get_legacy_name('tax_adjustment')), ),
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_tax_template
+        SET type_tax_use = 'adjustment'
+        WHERE type_tax_use = 'none' AND %s = TRUE
+        """, (AsIs(openupgrade.get_legacy_name('tax_adjustment')), ),
+    )
+
+
+def fill_account_chart_template_account_code_prefix(cr):
+    # if company_id was filled:
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_chart_template act
+        SET bank_account_code_prefix = rc.bank_account_code_prefix
+        FROM res_company rc
+        WHERE act.%s = rc.id AND act.bank_account_code_prefix IS NULL
+        """, (AsIs(openupgrade.get_legacy_name('company_id')), ),
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_chart_template act
+        SET cash_account_code_prefix = rc.cash_account_code_prefix
+        FROM res_company rc
+        WHERE act.%s = rc.id AND act.cash_account_code_prefix IS NULL
+        """, (AsIs(openupgrade.get_legacy_name('company_id')), ),
+    )
+    # if company_id was not filled:
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_chart_template act
+        SET bank_account_code_prefix = 'OUB'
+        WHERE act.bank_account_code_prefix IS NULL
+        """
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_chart_template act
+        SET cash_account_code_prefix = 'OUB'
+        WHERE act.cash_account_code_prefix IS NULL
+        """
+    )
+    # transfer_account_code_prefix:
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_chart_template act
+        SET transfer_account_code_prefix = trim(leading '0' from aat.code)
+        FROM account_account_template aat
+        WHERE act.%s = aat.id
+        """, (AsIs(openupgrade.get_legacy_name('transfer_account_id')), ),
+    )
+
+
+def update_res_company_account_setup_steps_states(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_setup_bank_data_state = 'done'
+        WHERE account_setup_bank_data_done = TRUE
+        """
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE res_company
+        SET account_setup_fy_data_state = 'done'
+        WHERE account_setup_fy_data_done = TRUE
```

Comment:
It's enough with `WHERE account_setup_fy_data_done`

Reply:
same as above